nbs/07_quickstart_for_web_devs.ipynb

```diff
@@ -85,7 +85,7 @@
    "source": [
     "What does that code do?\n",
     "\n",
-    "1. First we imported the FastHTML namespace on line 1. This is a carefully specified set of functions and other Python objects designed to optimize development of FastHTML projects. \n",
+    "1. In line 1, we import a carefully-specified set of FastHTML functions and other Python objects into our global namespace, for rapid development. \n",
```

Comment:
A little nitpick that we're importing objects rather than a namespace, but not sure if my wording is right here.